src/libclipper/include/clipper/metrics.hpp

```diff
@@ -291,6 +291,7 @@ class Histogram : public Metric {
   void insert(const int64_t value);
   const HistogramStats compute_stats();
   static double percentile(std::vector<int64_t> snapshot, double rank);
+  double percentile(double rank);
```

Comment:
Just calls `percentile(snapshot, rank)` with the same snapshot used in `compute_stats()`. It's just included because it might be convenient -- here's an example of its usage: https://github.com/nishadsingh1/clipper/blob/benchmark/src/benchmarks/src/noop_bench.cpp#L130.